app/views/issues/_attributes.rhtml

```diff
@@ -1046,6 +1051,17 @@ Event.observe(window, 'load', function() {
     }
   }
 
+  function updateFieldVisits() {
+    if (project && ["EcoATM", "Pokemon"].includes(project)) {
+      var visits = [].concat($$('label'))
```

Comment:
Is this specific enough to only select the things we want?

Reply:
yeah, this is specific enough to calculate visits (select all attributes and then filter `Dispatch Arrival`)